ddl/ddl.go

```diff
@@ -61,7 +62,9 @@ type DDL interface {
 	// all servers have the same lease time.
 	SetLease(lease time.Duration)
 	// Stat returns the DDL statistics.
-	Stat() (map[string]interface{}, error)
+	Stat() (map[string]*variable.StatusVal, error)
```

Comment:
`Stat()` gets a lot of statuses, and each status has own scope, so can't use `Stat()` to a unified set.